ngraph/frontend/onnx/frontend/src/editor.cpp

```diff
@@ -191,6 +192,14 @@ struct onnx_editor::ONNXModelEditor::Impl {
 
     Impl(const std::string& model_path)
         : m_model_proto{std::make_shared<ONNX_NAMESPACE::ModelProto>(onnx_common::parse_from_file(model_path))} {}
+
+    Impl(std::istream& model_stream)
+        : m_model_proto{std::make_shared<ONNX_NAMESPACE::ModelProto>(onnx_common::parse_from_istream(model_stream))} {}
+
+#if defined(ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
```

Comment:
Looks like `#if defined(ENABLE_UNICODE_PATH_SUPPORT)` should be enough

Reply:
Unfortunately, to open `std::ifstream` file stream with std::wstring as argument `ENABLE_UNICODE_PATH_SUPPORT` seems to be not enough